nilearn/plotting/img_plotting.py

```diff
@@ -556,6 +559,163 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
+            For display_mode == 'x', 'y', or 'z', then these are the
+            coordinates of each cut in the corresponding direction.
+            If None is given, the cuts is calculated automaticaly.
+            If display_mode is 'x', 'y' or 'z', cut_coords can be an integer,
+            in which case it specifies the number of cuts to perform
+        output_file: string, or None, optional
+            The name of an image file to export the plot to. Valid extensions
+            are .png, .pdf, .svg. If output_file is not None, the plot
+            is saved to a file, and the display is closed.
+        display_mode: {'ortho', 'x', 'y', 'z'}
+            Choose the direction of the cuts: 'x' - saggital, 'y' - coronal,
+            'z' - axial, 'ortho' - three cuts are performed in orthogonal
+            directions.
+        figure : integer or matplotlib figure, optional
+            Matplotlib figure used or its number. If None is given, a
+            new figure is created.
+        axes : matplotlib axes or 4 tuple of float: (xmin, ymin, width, height), optional
+            The axes, or the coordinates, in matplotlib figure space,
+            of the axes used to display the plot. If None, the complete
+            figure is used.
+        title : string, optional
+            The title displayed on the figure.
+        annotate: boolean, optional
+            If annotate is True, positions and left/right annotation
+            are added to the plot.
+        draw_cross: boolean, optional
+            If draw_cross is True, a cross is drawn on the plot to
+            indicate the cut plosition.
+        black_bg: boolean, optional
+            If True, the background of the image is set to be black. If
+            you wish to save figures with a black background, you
+            will need to pass "facecolor='k', edgecolor='k'" to pylab's
+            savefig.
+        cmap: matplotlib colormap, optional
+            The colormap for the atlas maps
+        vmin: float
+            Lower bound for plotting, passed to matplotlib.pyplot.imshow
+        vmax: float
+            Upper bound for plotting, passed to matplotlib.pyplot.imshow
+        alpha: float between 0 and 1
+            Alpha value sets the transparency of the color fillings inside the
+            contours.
+    """
+    display = plot_anat(anat_img, cut_coords=cut_coords,
+                        display_mode=display_mode,
+                        figure=figure, axes=axes, title=title,
+                        annotate=annotate, draw_cross=draw_cross,
+                        black_bg=black_bg, **kwargs)
+
+    maps_img = _utils.check_niimg_4d(maps_img)
+    n_maps = maps_img.shape[3]
+
+    _view_type = set(('auto', 'contours', 'filled_contours', 'continuous'))
+    if isinstance(view_type, _basestring):
+        if view_type not in _view_type:
+            raise TypeError('view_type option should be given '
```

Comment:
This should be a ValueError rather than a TypeError I reckon.

Also your error message is incomplete, 'auto' is another possibility that you haven't listed.